corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py

```diff
@@ -172,6 +172,9 @@ def ensure_roles(self, roles, dry_run=False):
         Role(slug=privileges.LINKED_PROJECTS,
              name='Linked Projects',
              description='Allows admin users to push and/or pull content between linked projects.'),
+        Role(slug=privileges.RELEASE_MANAGEMENT,
+             name='Release Management',
+             description='Allows access to features that help manage releasing content between projects.'),
```

Comment:
I might also mention linked projects in this description, since that's probably going to remain the most common internal way to refer to this feature.